multi-node/aws/cmd/kube-aws/command_destroy.go

```diff
@@ -33,6 +36,20 @@ func runCmdDestroy(cmd *cobra.Command, args []string) {
 
 	c := cluster.New(cfg, newAWSConfig(cfg))
 
+	if !force {
+		var confirmation string
+
+		fmt.Print("Are you sure you want to destroy the cluster? This action cannot be undone. Type \"yes\" to proceed: ")
```

Comment:
Simplify this to `Are you sure you want to permanently destroy the cluster? [y/N] `